R/income_tax.R

```diff
@@ -31,18 +31,18 @@
 #' @export income_tax
```

Comment:
In the roclets, please:

1. Add yourself as an author
2. Modify the entry for `fy.year` explaining the behaviour when `fy.year = NULL`.

Reply:
Just checking, that's the part at the top of the R code?